jgrapht-core/src/main/java/org/jgrapht/alg/util/NeighborIndex.java

```diff
@@ -44,11 +43,14 @@
  *
  * @author Charles Fry
  * @since Dec 13, 2005
+ * 
+ * @deprecated Use {@link org.jgrapht.alg.util.NeighborCache} instead.
  */
+@Deprecated
```

Comment:
Don't move this deprecated class; it should stay in the original location since that's where existing apps are referencing it.